packages/auth/src/features/authenticate/handler.test.ts

```diff
@@ -11,7 +11,7 @@
  */
 import * as fetchAny from 'jest-fetch-mock'
 import { createServerWithEnvironment } from '@auth/tests/util'
-import { createServer } from '@auth/index'
+import { createServer } from '@auth/server'
```

Comment:
Our current version of Jest doesn't support the "exports" field either. I tried updating jest and ts-jest, but that would've also required us to update TypeScript 😮‍💨  I changed all of our apps to never import the index file in tests. That's only for when the app is really run. Instead, test files now import a `server.ts` file that has the initialisation for server. All global side effects like monitoring initialisation can now be kept in `index.ts`. Seems to work fine and this is how I think I should've done it from the beginning 👍